index.js

```diff
@@ -0,0 +1,32 @@
+const slider=document.querySelector('.slider')
+const buttonNext=document.querySelector('.slider-button-next')
+const buttonPrev=document.querySelector('.slider-button-prev')
+const sheet= document.querySelector('.sheet')
+const sidebar=document.querySelector('.sidebar')
+const burgerMenu=document.querySelector('.burger-menu')
+const sidebarNav=document.querySelectorAll('.sidebar-nav-list__component')
+const sideBarBurgerButton=document.querySelector('.sidebar-button')
+function toggleSlider(e){
+
+        slider.classList.toggle('slider-2')
+        slider.classList.toggle('slider-1')
+
+}
+function hideSidebar(){
+        sheet.style.display='none'
```

Comment:
try to avoid changing styles through 'style'. Add 2 classes, like "stateOn", "stateOff" (these are examples) and add/remove these classes through js

Reply:
Done